OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardFragment.java

```diff
@@ -297,16 +439,31 @@ private void fetchCourseById() {
             protected void onResponse(@NonNull final EnrolledCoursesResponse course) {
                 if (getActivity() != null) {
                     getArguments().putSerializable(Router.EXTRA_COURSE_DATA, course);
-                    UiUtils.INSTANCE.restartFragment(CourseTabsDashboardFragment.this);
+                    FullscreenLoaderDialogFragment fullscreenLoader = FullscreenLoaderDialogFragment
```

Comment:
Can we write a separate method to get `fullscreenLoader` as we are accessing it multiple times

Reply:
We already use a different method within the `FullscreenLoaderFragment` class to fetch the instance. 🤔 If you meant something else please let me know.

Reply:
we can call getFullscreenLoader() everytime we need to check its visibility.
```
    private FullscreenLoaderDialogFragment getFullscreenLoader(){
        return FullscreenLoaderDialogFragment.getRetainedInstance(getChildFragmentManager());
    }